module/modmanager/manager_test.go

```diff
@@ -1001,7 +1001,7 @@ func greenLog(t *testing.T, msg string) {
 
 func TestRTPPassthrough(t *testing.T) {
 	ctx := context.Background()
-	logger := logging.NewTestLogger(t)
+	logger := logging.NewInMemoryLogger(t)
```

Comment:
Oh, this is a neat idea! IIUC though, this change results in the test never logging anything. Not a problem when the test passes, but a bummer if it fails. I think we have two options here:
- `defer logger.OutputLogs()` which will always output test logs at the end of the test (pass or fail). But because that gets a copy of logs up until that point, and logs them before go marks the test as "done", everything will be fine
- do the `defer func() { if test failed: logger.OutputLogs }()`. So a passing test never logs anything. And a failing test will log its output, but similarly won't result in any "log after test finished" errors.

Reply:
did the latter